pkg/serviceaccount/externaljwt/plugin/testing/v1/externalsigner_mock.go

```diff
@@ -57,6 +57,8 @@ type MockSigner struct {
 	FetchError    error
 	MetadataError error
 	errorLock     sync.RWMutex
+
+	v1.UnimplementedExternalJWTSignerServer
```

Comment:
prefer `UnsafeExternalJWTSignerServer` so we get notified if a new method gets added we need to implement?

Reply:
Makes sense, fixed as suggested.